vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -161,6 +161,12 @@ bool TextViewManager::UpdateProperty(
   } else if (TryUpdateTextAlignment(textBlock, propertyName, propertyValue)) {
   } else if (TryUpdateTextTrimming(textBlock, propertyName, propertyValue)) {
   } else if (TryUpdateTextDecorationLine(textBlock, propertyName, propertyValue)) {
+    // Temporary workaround for bug in XAML which fails to flush old TextDecorationLine render
```

Comment:
Link to the bug issue:
https://github.com/microsoft/microsoft-ui-xaml/issues/1093#issuecomment-514282402

Reply:
Lets make sure to open an issue in this repo to remove this work around once when xaml bug is fixed.